components/dashboard/src/App.tsx

```diff
@@ -279,6 +279,12 @@ function App() {
         window.location.host = "www.gitpod.io";
         return <div></div>;
     }
+    
+    // redirect for Anton
+    if (isGitpodIo() && && window.location.pathname === "/akosyakov") {
```

Comment:
TODO: Improve JavaScript skillz:

```suggestion
    if (isGitpodIo() && window.location.pathname === "/akosyakov") {
```